Cars/Model/Employee.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Cars.Model
+{
+    public record Employee
+    {
+        public static int lastId = 0;
```

Comment:
lastId cannot be here, when we will be connecting to the mySQL DB, migration will create wrong Table, with lastId in it